app/uk/gov/hmrc/taxhistory/model/nps/IabdType.scala

```diff
@@ -123,9 +126,31 @@ case class Iabd(nino: String,
                 `type`: IabdType,
                 grossAmount : Option[BigDecimal] = None,
                 typeDescription : Option[String] = None,
-                source: Option[Int] = None) {
-
-  def toStatePension = StatePension(grossAmount = grossAmount.getOrElse(0.0), typeDescription.getOrElse(""))
+                source: Option[Int] = None,
+                paymentFrequency: Option[Int],
+                startDate: Option[String]) {
```

Comment:
can we assign `None` by default for these new fields ?

Reply:
Yep, done.